dev-libs/opensp/opensp-1.5.2.recipe

```diff
@@ -0,0 +1,96 @@
+SUMMARY="A C++ library for using SGML/XML files"
+DESCRIPTION="An SGML Extended Facilities system conforming to Annex A of International Standard ISO/IEC 10744 -- Hypermedia/Time-based Structuring Language"
+HOMEPAGE="https://openjade.sourceforge.net/"
+COPYRIGHT="1994, 1995, 1996, 1997, 1998 James Clark"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/openjade/OpenSP-$portVersion.tar.gz"
+CHECKSUM_SHA256="57f4898498a368918b0d49c826aa434bb5b703d2c3b169beb348016ab25617ce"
+SOURCE_DIR="OpenSP-$portVersion"
+PATCHES="opensp-1.5.2.patchset"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+libVersion="5.0.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	opensp$secondaryArchSuffix = $portVersion
+	lib:libosp$commandSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	opensp${secondaryArchSuffix}_devel = $portVersion
+	devel:libosp$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	opensp$secondaryArchSuffix == $portVersion base
+	"
+
+SUMMARY_tools="Command line tools for opensp"
+PROVIDES_tools="
+	opensp${secondaryArchSuffix}_tools = $portVersion
+	cmd:onsgmls$commandSuffix
+	cmd:osgmlnorm$commandSuffix
+	cmd:ospam$commandSuffix
+	cmd:ospcat$commandSuffix
+	cmd:ospent$commandSuffix
+	cmd:osx$commandSuffix
+	"
+REQUIRES_tools="
+	haiku$secondaryArchSuffix
+	opensp$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:automake
+	cmd:autoreconf
+	cmd:gcc$secondaryArchSuffix
+	cmd:gettext
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:perl
```

Comment:
Missing cmd:bison, if you enable the documentation/manpages cmd:xmlto is also required.

Reply:
If manpages/documentation are enabled, manpages should go to the tools package, the documentation to the devel package.

Reply:
I don't even have bison installed on my system, do you mean it would be a requirement for the manpages?
Is it building the documentation recommended or is it "up to me"?

Reply:
From a quick look in the source I could find some relation to libintl.

Reply:
Also, if you give a closer look at when configure is running you will see it is checking for these, they don't fail when not found, so I gather it will use the included version.